doc/index.rst

```diff
@@ -147,50 +148,26 @@ PennyLane is **free** and **open source**, released under the Apache License, Ve
    concepts/autograd_quantum
    zreferences
 
-
 .. toctree::
    :maxdepth: 1
-   :caption: Tutorials
+   :caption: Quickstart
```

Comment:
The names of these entries are all TBD, and are up for discussion in this PR (most of the names I picked are placeholders)

Reply:
I think this sounds good. One question: When we add my PRs on the tutorials, it had more subsections than just "Tutorials". I had "Advanced usage" and "QML algorithms" in separate sections. Would that be okay or should we keep the single page in the TOC? It might become a really long page though...

Reply:
My preference is to have it in one page (for now), and split it up into multiple pages when we have more tutorials. Just to avoid each tutorial page looking a bit bare.

But this is open to discussion here!

Reply:
I think one page is doable. only the advanced usage section has a lot more text and code snippets. I can see if I can restructure it to have thumbnails only. But we still get a lot of headings on this  one page : basic tutorials, interfaces tutorials, plugins tutorials, advanced usage with 2-3 subheadings, QML algorithms.

Maybe I try this and see how it looks?

Reply:
Also, having a separate TOC for QML algorithms will be good for non-beginner users looking for advanced examples and not basic tutorials. They might not see these algorithm implementations if they think they do not need to go to the tutorials page. 

Moreover, this section might grow as we add more advanced problems implemented in PennyLane here.

Reply:
Should I do that in a separate PR so we can go ahead with this PR in the meantime?